### ISSUE TITLE
Zoom disabled on the Contact page

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1619]
The Contact page includes a ‘<meta>’ tag with the name ‘viewport’ and the attribute ‘user-scalable=no’, which disables the ability to zoom in on mobile devices. This prevents users with visual impairments from enlarging the content to make it more readable. To resolve this problem, the ‘user-scalable=no’ attribute should be removed.
